crux-core/src/crux/api/alpha/EntityTxWithDocument.java

```diff
@@ -0,0 +1,31 @@
+package crux.api.alpha;
+
+import clojure.lang.Keyword;
+
+import java.util.Map;
+
+public class EntityTxWithDocument {
```

Comment:
hmm :thinking:

Reply:
Agreed on the hmm - was unsure if I should just edit what I had with EntityTx, or return some sort of tuple (as I did here, sort of)